spec.bs

```diff
@@ -2015,6 +2002,15 @@ To <dfn>convert to an AuctionAd sequence</dfn> given a [=list=]-or-null |ads|:
     1. Let |adIDL| be a new {{AuctionAd}}.
     1. [=map/Set=] |adIDL|["{{AuctionAd/renderURL}}"] to the [=URL serializer|serialization=] of
       |ad|'s [=interest group ad/render url=].
+    1. If |ad|'s [=interest group ad/selectable buyer and seller reporting IDs=] is not null:

```

Comment:
Yeah, it may sound weird but they are in fact passed. 

It's clarified more [in the developer guide for deals](https://developers.google.com/privacy-sandbox/private-advertising/protected-audience-api/use-case/deals#2_deal_and_seat_ids_for_buyers_during_bid_generation)

Reply:
I see. thanks